repos/fdbt-site/src/pages/viewPassengerTypes.tsx

```diff
@@ -171,20 +176,28 @@ const PassengerTypeGroups = ({ passengerTypeGroups }: { passengerTypeGroups: Gro
                                     </ul>
                                 </div>
 
-                                <h4 className="govuk-!-padding-bottom-4">{sentenceCaseString(d.name)}</h4>
+                                <h4 className="govuk-!-padding-bottom-4">
+                                    {sentenceCaseString(passengerTypeGroup.name)}
+                                </h4>
 
                                 <p className="govuk-body-s govuk-!-margin-bottom-2">
-                                    <span className="govuk-!-font-weight-bold">Max size:</span> {d.maxGroupSize}
+                                    <span className="govuk-!-font-weight-bold">Max size:</span>{' '}
+                                    {passengerTypeGroup.maxGroupSize}
                                 </p>
 
-                                {d.companions.map((c) => (
-                                    <p key={c.passengerType} className="govuk-body-s govuk-!-margin-bottom-2">
-                                        <span className="govuk-!-font-weight-bold">
-                                            {sentenceCaseString(c.passengerType)}:
-                                        </span>{' '}
-                                        {c.minNumber ? c.minNumber : '0'} - {c.maxNumber}
-                                    </p>
-                                ))}
+                                {passengerTypeGroup.companions.length
```

Comment:
I think this was what was causing the error @RobCatton. There was no check to ensure `companions` array wasn't empty